Backpack/build.gradle

```diff
@@ -32,9 +31,11 @@ android {
     ]
   }
   lintOptions {
+    baseline file("lint-baseline.xml")
     lintConfig file("../lint.xml")
     checkReleaseBuilds false // we're already running lint separately
     warningsAsErrors true
+    disable 'UnusedResources' // we're exposing resources for consumers
```

Comment:
not sure why this didn't trigger before, but we don't care about unused resources here as we're exposing things for the app that we don't use in the library